test/helpers/runtime.go

```diff
@@ -21,5 +21,9 @@ import (
 //CreateNewRuntimeHelper returns Docker and Cilium helpers for running the
 //runtime tests on the provided VM target and using logger log .
 func CreateNewRuntimeHelper(target string, log *log.Entry) (*Docker, *Cilium) {
-	return CreateDocker(target, log), CreateCilium(target, log)
+	log.Infof("creating docker")
+	docker := CreateDocker(target, log)
```

Comment:
I'm a little surprised that these log messages are being added in `CreateNewRuntimeHelper()`, and not `CreateDocker()`.

Reply:
I'll remove them. I have a plan to remove the concept of "Docker" and "Cilium" helpers in a follow-up PR, as the different types don't provide much value and have identical members.